SingularityUI/app/components/dashboard/MyPausedRequests.jsx

```diff
@@ -13,13 +13,13 @@ const MyPausedRequests = ({userRequests}) => {
     <div className="empty-table-message"><p>No paused requests</p></div>
   );
 
-  const pausedRequests = userRequests.filter((r) => r.state === 'PAUSED');
+  const pausedRequests = userRequests.filter((request) => request.state === 'PAUSED');
 
   if (pausedRequests.length > 0) {
     pausedRequestsSection = (
       <UITable
         data={pausedRequests}
-        keyGetter={(r) => r.request.id}
+        keyGetter={(request) => request.request.id}
```

Comment:
Yeah, I'm realizing that. I'm willing to go through and change them all - it's probably worth doing.